README.md

```diff
@@ -1,5 +1,5 @@
 # Safecast/safecastapi
-[![Build Status](https://travis-ci.org/Safecast/safecastapi.svg?branch=master)](https://travis-ci.org/Safecast/safecastapi) [![Code Climate](https://codeclimate.com/github/Safecast/safecastapi/badges/gpa.svg)](https://codeclimate.com/github/Safecast/safecastapi) [![Test Coverage](https://codeclimate.com/github/Safecast/safecastapi/badges/coverage.svg)](https://codeclimate.com/github/Safecast/safecastapi/coverage)
+[![Build Status](https://travis-ci.org/Safecast/safecastapi.svg?branch=master)](https://travis-ci.org/Safecast/safecastapi) [![Code Climate](https://codeclimate.com/github/Safecast/safecastapi/badges/gpa.svg)](https://codeclimate.com/github/Safecast/safecastapi) [![Test Coverage](https://codeclimate.com/github/Safecast/safecastapi/badges/coverage.svg)](https://codeclimate.com/github/Safecast/safecastapi/coverage) <a href="https://zenhub.com"><img src="//dxssrr2j0sq4w.cloudfront.net/3.2.0/img/external/zenhub-badge.png" alt="ZenHub logo"></a>
```

Comment:
Should this include `https`? Doesn't seem like it's rendering

![Screen Shot 2019-05-23 at 3 04 51 PM](https://user-images.githubusercontent.com/690/58229271-2dcf2180-7d6c-11e9-9b9d-50ddb3199333.png)

Also can we link to https://app.zenhub.com/workspaces/safecastapi-5c91af1f7a7e856312dbd3b7/board ? That's the board representation I use for the weekly.

Reply:
Changed. GitHub markdown parser seems not to like URI without schema.